interactions.go

```diff
@@ -63,17 +64,22 @@ type InteractionType uint8
 
 // Interaction types
 const (
-	InteractionPing = InteractionType(iota + 1)
+	InteractionPing InteractionType = iota + 1
 	InteractionApplicationCommand
+	InteractionButton
```

Comment:
This should be `InteractionMessageComponent`, and you will have to check elsewhere that the Message Component type is `Button`.

Reply:
> This should be `InteractionMessageComponent`, and you will have to check elsewhere that the Message Component type is `Button`.

Oh, alright.